### PR TITLE
Added log4j-slf4j dependencies

### PR DESCRIPTION
**Pull request by slammy-dev**
Added log4j2.properties, added dependencies in pom.xml, log folder excluded in git (.gitignore)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,5 @@ docs
 *.iml
 /target
 ~*
-*.lnk
\ No newline at end of file
+*.lnk
+/test
```
Comment (SMishAlex) [inline .gitignore]: what is the test directory?

File: pom.xml [hunk 2]
```diff
@@ -64,8 +64,24 @@
...
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j-impl</artifactId>
+      <version>2.1</version>
+    </dependency>
+
   </dependencies>
 
+
+
+
```
Comment (SMishAlex) [inline pom.xml]: what are those spaces for?

File: .gitignore (hunk 1 above)
Comment (slammy-dev) [reply .gitignore]: I screwed up. I wanted to all /logs dirrectory

File: pom.xml (hunk 2 above)
Comment (slammy-dev) [reply pom.xml]: removed.

**Pull request merged**